src/tests/functional/test_competitions.py

```diff
@@ -27,6 +37,7 @@ def _upload_competition(self, competition_zip_path):
         comp_url = reverse("competitions:detail", kwargs={"pk": comp.id})
         self.find(f'a[href="{comp_url}"]').click()
         self.assert_current_url(comp_url)
+        self.current_server_time_exists()
```

Comment:
this test should be run as a standalone, not a part of the upload competitions step. If this test is failing, we should not have to re-run this _very_ long running test.

things that should be tested would be: 
- server time visible if anonymous user
- server time visible if logged in